src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -327,14 +329,14 @@ private boolean confirmOverwriteFiles() {
     gbc.anchor = GridBagConstraints.EAST;
     container.add(pemFileLabel, gbc);
     startDateLabel = new JLabel();
-    startDateLabel.setText("Start date");
+    startDateLabel.setText("Start date (inclusive)");
```

Comment:
In this file, remove all changes except this one and the next

Reply:
Which changes are you talking about here?